tests/queries/0_stateless/01052_window_view_proc_tumble_to_now.sql

```diff
@@ -0,0 +1,18 @@
+SET allow_experimental_window_view = 1;
+
+DROP TABLE IF EXISTS mt;
+DROP TABLE IF EXISTS dst;
+DROP TABLE IF EXISTS wv;
+DROP TABLE IF EXISTS `.inner.wv`;
+
+CREATE TABLE dst(count UInt64) Engine=MergeTree ORDER BY tuple();
+CREATE TABLE mt(a Int32) ENGINE=MergeTree ORDER BY tuple();
+CREATE WINDOW VIEW wv TO dst AS SELECT count(a) AS count FROM mt GROUP BY TUMBLE(now(), INTERVAL '1' SECOND, 'US/Samoa') AS wid;
+
+INSERT INTO mt VALUES (1);
+SELECT sleep(2);
```

Comment:
This test looks flaky - https://clickhouse-test-reports.s3.yandex.net/32080/f416815121beaaaedecb165157d92f4db5764151/fast_test.html#fail1

Reply:
The process in `processing time` might not be deterministic because the window is triggered by the wall time of the machine. For a tumbling window with 3 seconds size as follows:
```
|           |           |
|___|___|___|___|___|___|
0   1   2   3   4   5   6
```
**Case 1**: If data is inserted at 2.9999s, but when it arrives at the window view, the wall time is 3.0001s, this data will discarded because the first window is already triggered(Do not consider `ALLOWED_LATENESS`). The probability of this happening is extremely low, but I’m not sure if we can ensure it will not happen.

**Case 2**: If we use different timezones in a query, like `TUMBLE(now('US/Samoa'), x, x)`. The tumble function might return a DataTime one day ago, so that the newly inserted data will be classified into the window that has been already triggered, which also causes loss data. (This case can be solved by using timezone when fetching wall time)

Reply:
I believe the problem with flacky test might be because we do not consider timezone here:
https://github.com/ClickHouse/ClickHouse/blob/7de34bb8e16bcf6dd2e6c995f0b7cd124abb492b/src/Storages/WindowView/StorageWindowView.cpp#L1167-L1169

Reply:
I'll fix this ASAP.

Reply:
> I'll fix this ASAP.

Thanks, but no need now, I decided to fix myself https://github.com/ClickHouse/ClickHouse/pull/32178 :)